### PR TITLE
Compression

### PR DESCRIPTION
**Pull request by jburel**
Make sure the correct compression level is used when opening the viewer

To test the PR
 * Open an image. Modify the compression level e.g. set it to high
 * Close the viewer
 * Open another image, Modify the compression level e.g. to none. Check that the image is updated.

Problem noticed while working on https://github.com/openmicroscopy/openmicroscopy/pull/3368

### CODE REVIEW

File: components/insight/SRC/org/openmicroscopy/shoola/agents/imviewer/ImageLoader.java [hunk 1]
```diff
@@ -69,7 +69,10 @@
     
     /** Indicates that the rendering of that image has been cancelled.*/
     private boolean cancelled;
-    
+
+    /** The compression level to use.*/
```
Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/agents/imviewer/ImageLoader.java]: update copyright header?

**Comment by jburel** [comment]
@mtbc: you could test with the following image
 * user-3 read-annotate-1
 * First image in archivedv dataset i.e. /home/hudson/for_import_to_howe/archived-imoprt-case/user-3/read-annotate-1/no_projects/archivedv/CFPNEAT01_R3D.dv.log

**Comment by mtbc** [comment]
Thank you, that's a great help. Even for not-big images, it really does seem to be compressing the image according to what bandwidth option I set on login or switch user. Good to merge.

**Comment by jburel** [comment]
@mtbc: done

**Comment by mtbc** [comment]
Latest changes good. If Travis is happy (hah!) then so am I.

**Comment by jburel** [comment]
--rebased-to #3406

**Pull request merged**